src/sentry/conf/server.py

```diff
@@ -2204,15 +2204,41 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
     "event:read",
     "event:write",
     "event:admin",
-    "alerts:write",
     "alerts:read",
+    "alerts:write",
     # openid, profile, and email aren't prefixed to maintain compliance with the OIDC spec.
     # https://auth0.com/docs/get-started/apis/scopes/openid-connect-scopes.
     "openid",
     "profile",
     "email",
 }
 
+SENTRY_SCOPE_HIERARCHY_MAPPING = {
```

Comment:
do we have a list of valid scopes anywhere you can do a check against? if somebody adds a new scope they may forget to add it here

Reply:
This [new test](https://github.com/getsentry/sentry/pull/54510/files#diff-4b5c8a9f0cc8cbdf7bccc92f465beb16a38fc51a560d0333881fce526fc81949R6) checks that the mapping contains all scopes

Reply:
is it possible to generate the mapping dynamically from the list of scopes?

Reply:
You can generally for `read/write/admin` scopes, but there unique scopes like `org:integrations` that need to be added `org:admin`. I'd prefer to keep a data structure of the scopes to avoid these edge cases and make it easy/human-readable without having to dig through code.